src/Intl/FranceCitiesBundle.php

```diff
@@ -36,6 +36,24 @@ public static function getCity(string $postalCode, string $inseeCode)
         return self::$cities[$postalCode][$inseeCode] ?? null;
     }
 
+    /**
+     * Returns the cities indexed by INSEE code.
+     */
+    public static function getCityByInseeCode(): array
+    {
+        static $citiesByInsee = [];
```

Comment:
why `static` ?

Reply:
this is to keep the values between multiple calls. this works like a class property scoped to the function. see : https://www.php.net/manual/fr/language.variables.scope.php#language.variables.scope.static